c++/src/capnp/encoding-test.c++

```diff
@@ -1667,6 +1667,8 @@ TEST(Encoding, Embeds) {
     checkTestMessage(reader.getRoot<TestAllTypes>());
   }
 
+#if !CAPNP_LITE
```

Comment:
A clearer explanation for this commit would be: Stringification is not available in lite mode, so skip test that relies on it.

(Stringification depends on the dynamic API which by definition is not available in lite mode.)